VideoLocker/res/layout/row_discussion_thread.xml

```diff
@@ -30,28 +33,35 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:textColor="@color/edx_grayscale_neutral_x_dark"
-            android:textSize="@dimen/edx_small" />
+            android:textSize="@dimen/edx_small"
+            android:maxLines="2"
+            android:ellipsize="end"
+            tools:text="This is a post title. It is a very long title to demonstrate that post titles will be truncated when they are too long."/>
 
         <LinearLayout
             android:orientation="horizontal"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content">
 
-            <org.edx.mobile.third_party.iconify.IconView
-                android:id="@+id/discussion_post_pin_or_following_icon"
-                style="@style/icon_view_standard_size_style"
-                android:layout_marginEnd="3dp"
-                android:layout_marginRight="3dp"
-                app:iconColor="@color/edx_grayscale_neutral_light"
-                android:visibility="gone" />
+            <TextView
+                android:id="@+id/discussion_post_pin_text_view"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:drawablePadding="3dp"
```

Comment:
Should this be exported as a globally reusable dimen (and maybe add it in a `edX.Widget.TextView` style)? The same value is being used in the next `TextView` as well.